src/vstart.sh

```diff
@@ -1004,14 +1004,20 @@ do_rgw()
         RGWDEBUG="--debug-rgw=20"
     fi
 
+    CEPH_RGW_HTTPS="${CEPH_RGW_PORT: -1}"
+    if [[ ${CEPH_RGW_HTTPS} = "s" ]]; then
+        CEPH_RGW_PORT="${CEPH_RGW_PORT::-1}"
```

Comment:
I suggest adding a new variable for example CEPH_RGW_PORT_NUM and using it for the check and printing
 I prefer you did not change CEPH_RGW_PORT